Phonics/Content Model/PHContent.swift

```diff
@@ -111,7 +116,26 @@ class PHContentManager {
                     quizWords = quizWordsArray.compactMap{ wordForString($0.trimmingWhitespace()) }
                 }
                 
-                let sound = Sound(sourceLetter: letter,
+                //parse color for each Sound
+                let colorString = line[7].trimmingWhitespace()
+                var color: UIColor
+                switch colorString {
+                case "orange":
+                    color = .orange
+                case "green":
+                    color = .green
```

Comment:
This is the one to change. `.green` is a bit too bright. Probably will need to be a custom shade of green